src/main/java/sysc4806/graduateAdmissions/model/Term.java

```diff
@@ -13,14 +15,17 @@
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
+@Builder
 @Entity
 public class Term {
 
     @Id
-    @GeneratedValue
-    private long id;
-    private boolean active;
-    private String deadline;
+    @GeneratedValue(strategy=GenerationType.IDENTITY)
+    private Long id;
+    private Boolean active;
+    @Temporal(TemporalType.DATE)
+    @DateTimeFormat(pattern = "yyyy.MM.dd")
+    private Date deadline;
     @Enumerated(EnumType.STRING)
     private Season season;
     private String year;
```

Comment:
I know someone else made this entity, and this isn't necessary right now, but it might make more sense for the year field to be an int, so we can compare it to the the year in deadline or the current date more easily